RELEASING.md

```diff
@@ -78,8 +78,9 @@ git checkout 3.8-customer-replica
 # Create the new pure-docker release branch
 git checkout -B 3.9-customer-replica
 
-# Merge the publish-3.9 branch, which will have been created by the release tool, into the pure-docker release branch. Default to taking the remote changes. Note that "theirs" is a Git term, not a placeholder.
-git merge publish-3.9 -X theirs
+# Merge the publish-3.9.0 branch, which will have been created by the release tool, into the pure-docker release branch. Default to taking the remote changes. Note that "theirs" is a Git term, not a placeholder.
+# Note the publish-x.x.x branch is usually in the format `publish-x.x.x`
```

Comment:
I'm not sure how saying "publish-x.x.x" is usually in the form of "publish-x.x.x" is clear enough for the intention (that I understand) of this PR?

I think the updated example of "publish-3.9.0" is good enough.

Reply:
Oh! I can make the change to `publish-3.9.0`, the main cause of confusion for me is that the documentation says `publish-3.9` instead of `publish-3.9.0` and that confused me.

Reply:
This is just a comment to let the reader not be confused, the actual command was actually updated to use `publish-3.9.0`